src/Microsoft.Azure.WebJobs.Host/Scale/ScaleManager.cs

```diff
@@ -0,0 +1,295 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Host.Scale
+{
+    public class ScaleManager
```

Comment:
For any new public types we expose, we'll need to have good xml doc on classes/methods